docs/pages/database-access/auto-user-provisioning.mdx

```diff
@@ -5,9 +5,11 @@ description: Configure automatic user provisioning for databases.
 
 (!docs/pages/includes/database-access/auto-user-provisioning/intro.mdx!)
 
+For details about database object permissions feature see [here](./auto-user-provisioning/database-object-permissions.mdx).
```

Comment:
I would connect this paragraph to the previous one and indicate what database object permissions entail. Otherwise, I'm not sure readers will know what this means and whether this page is for them.

It's also good practice to add more descriptive link text than "here", e.g., the title of the page we link to, so the user doesn't need to mouse over the link to know what the target is.